packages/contracts-bedrock/test/Helpers.sol

```diff
@@ -1,33 +1,6 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import { ERC20 } from "@rari-capital/solmate/src/tokens/ERC20.sol";
-import { ERC721 } from "@rari-capital/solmate/src/tokens/ERC721.sol";
-import { ECDSA } from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import { OptimistInviter } from "src/periphery/op-nft/OptimistInviter.sol";
-import { IERC1271 } from "@openzeppelin/contracts/interfaces/IERC1271.sol";
-import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
-import { ECDSAUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/cryptography/ECDSAUpgradeable.sol";
-import { AdminFaucetAuthModule } from "src/periphery/faucet/authmodules/AdminFaucetAuthModule.sol";
-
-contract TestERC20 is ERC20 {
-    constructor() ERC20("TEST", "TST", 18) { }
-
-    function mint(address to, uint256 value) public {
-        _mint(to, value);
-    }
-}
-
-contract TestERC721 is ERC721 {
-    constructor() ERC721("TEST", "TST") { }
-
-    function mint(address to, uint256 tokenId) public {
-        _mint(to, tokenId);
-    }
-
-    function tokenURI(uint256) public pure virtual override returns (string memory) { }
-}
-
 contract CallRecorder {
```

Comment:
Looks like theres some stuff in this file that should have been deleted?

`CallRecorder` and `Reverter` are also present in `packages/contracts-bedrock/test/mocks/Callers.sol`

Reply:
Ah good catch i messed up my `:w`